java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/CompositeJdbcConsumer.java

```diff
@@ -74,3 +87,26 @@ public void resetVectorSchemaRoot(VectorSchemaRoot root) {
     }
   }
 }
+
+/**
```

Comment:
I'm not sure where to add this exception. It doesn't seem like there's any custom exceptions. I was thinking `java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/exceptions/JdbcConsumerException.java`?